js/chat_app/src/LoginPage.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { useState } from 'react';
+import LoginIcon from '@mui/icons-material/Login';
+import Grid from '@mui/material/Grid';
+import FormControl from '@mui/material/FormControl';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+
+export function LoginPage({ setUserData }) {
+  const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+
+  function submitLogin() {
+    if (userName && password) {
+      request(userName.trim(), password.trim()).then((result) =>
+        setUserData(result)
+      );
+
+      setUserName('');
+      setPassword('');
+    }
+  }
+
+  return (
+    <Grid container spacing={2} mt={5}>
+      <Grid item xs={12}>
+        <FormControl>
+          <Typography variant='h5' mb={5}>
+            Welcome to chat
+          </Typography>
+          <Box
+            component='form'
+            sx={{ display: 'flex', flexDirection: 'column', gap: 2 }}
+            noValidate
+            autoComplete='off'
+          >
+            <TextField
+              value={userName}
+              placeholder='Username'
+              onChange={(e) => setUserName(e.target.value)}
+            />
+            <TextField
+              value={password}
+              type='password'
+              placeholder='Password'
+              onChange={(e) => setPassword(e.target.value)}
+            />
+          </Box>
+          <Button
+            sx={{ mt: 3 }}
+            endIcon={<LoginIcon />}
+            onClick={() => submitLogin()}
```

Comment:
With forms, usually, we use` button` with `type="submit"` and handler onSubmit={submitLogin}
Now, as I don't see any differences in functionality between these two options, it will be easier to handle a form with the second variant when you add, for example, validation or loader on form submission.